recipes/dotty-dict/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "dotty-dict" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  pypi_name: "dotty_dict"
```

Comment:
This is going to need to be up at the top using the `set` syntax for the jinja templating.